pickle-lib/build.gradle

```diff
@@ -1,3 +1,8 @@
+plugins {
+    id 'java-gradle-plugin'
```

Comment:
Is this needed?

Reply:
This is trying to turn `pickle-lib` into a gradle plugin

Reply:
Oh I see what happened here, I copy-pasted the plugins block from `pickle-plugin`, thanks!

Reply:
e842e1f